charts/hono/templates/NOTES.txt

```diff
@@ -13,9 +13,21 @@ You can check the current status by issuing the following command:
 The output should look similar to this:
```

Comment:
The NOTES.txt still contains a link to https://eclipse.org/hono/getting-started-kafka/. That page doesn't exist any more.

And, while at it, you could also replace `service-device-registry-0` with something like `service-device-registry-64d645dd9c-q9c5q` below (no statefulset being used for device registry component).

Reply:
That depends on which type of registry you are using. The default (embedded) registry does indeed use a StatefulSet ...
But I can use a name reflecting the type of registry being installed ...

Reply:
Oh, right, I only checked with the mongoDB and jdbc registries.